### PR TITLE
Update USwitch component

### LINKED ISSUE #108: USwitch player JS only works with specific MIT domains
The USwitch player throws a JS error unless the domain from which it is loaded is localhost, mit.edu, or amps-web.amps.ms.mit.edu

The company which produces the player is working on modifications to fix this.

### PR DESCRIPTION
**Pull request by mbertrand**
#### What are the relevant tickets?
Fixes #108
Depends on https://github.com/mitodl/omniplayer/pull/1

#### What's this PR do?
Updates front-end code to work with the latest version of USwitch, which works with any .mit.edu subdomain.

#### How should this be manually tested?
- Set `USWITCH_URL=https://s3.amazonaws.com/odl-video-service-uswitch-dev/dev`
- Change a video to `multiangle=true` in the admin console
- Temporarily modify your hosts file so that 'video.odl.mit.edu' points to your docker IP
- Load that video's detail page.  The video should play with the USwitch player.

### CODE REVIEW

File: static/js/components/USwitchPlayer.js [hunk 1]
```diff
@@ -60,7 +60,9 @@ export default class USwitchPlayer extends React.Component {
 
   render() {
     return (
-        <div ref={ node => this.videoNode = node }  id='omniPlayer'></div>
+        <div ref={ node => this.videoNode = node }  allowFullScreen id='omniPlayer'>
+          <video id='video'></video>
+        </div>
```
Comment (giocalitri) [inline static/js/components/USwitchPlayer.js]: do we have tests for REACT?

**Comment by giocalitri** [comment]
@noisecapella sorry I had not seen you assigned yourself

**Comment by noisecapella** [comment]
It's fine, I took it because you were out for the day but if you want to review this feel free to take it back

**Comment by giocalitri** [comment]
I had just one comment: the functionality is fine.
Not sure you have more

**Comment by mbertrand** [comment]
@giocalitri @noisecapella I just updated the test for this.

**Pull request merged**